assets/dashboards/ohm-prometheus.json

```diff
@@ -550,18 +587,21 @@
         "text": {},
         "textMode": "auto"
       },
-      "pluginVersion": "8.4.0",
+      "pluginVersion": "9.3.1",
       "targets": [
         {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
           "exemplar": true,
-          "expr": "sum(increase(ohm_hdd_bytes{instance=\"$instance\", sensor=\"Data Written\"}[$__range]))",
-          "hide": false,
+          "expr": "sum(increase(ohm_nic_bytes{instance=\"$instance\",sensor=\"Data Uploaded\"}[$__range]))",
           "interval": "",
           "legendFormat": "{{sensor}}",
-          "refId": "B"
+          "refId": "A"
         }
       ],
-      "title": "Disk Writes",
+      "title": "Uploads",
```

Comment:
Looks like the Disk Writes and Uploads panels were swapped, can you swap them back?